download/release-notes-0.9.0.md

```diff
@@ -0,0 +1,253 @@
+---
+title: Release Notes 0.9.0
+sidebar_position: 995
+---
+
+# Uniffle Release 0.9.0
+
+## Highlight
+
+- Introduce dashboard.
+- Introduce rust-based shuffle server.
```

Comment:
Your work about blockId should be highlight.
I would like to add `The data transportation in the Netty mode became production available`.
And remove `Add dynamic allocation patch for Spark` and `Add shaded spark client`.

Reply:
I have removed the two bullet points and added the new one, paraphrased:

    The data transportation Netty mode is production available.